patches/minecraft/net/minecraft/world/gen/layer/BiomeLayer.java.patch

```diff
@@ -0,0 +1,75 @@
+--- a/net/minecraft/world/gen/layer/BiomeLayer.java
++++ b/net/minecraft/world/gen/layer/BiomeLayer.java
+@@ -28,12 +28,32 @@
+    private static final int[] field_202746_t = new int[]{field_202730_d, field_202729_c, field_202741_o, field_202736_j};
+    private static final int[] field_202747_u = new int[]{field_202731_e, field_202731_e, field_202731_e, field_202742_p};
+    private int[] field_151623_c = field_202744_r;
++   private java.util.List<net.minecraftforge.common.BiomeManager.BiomeEntry>[] biomes = new java.util.ArrayList[net.minecraftforge.common.BiomeManager.BiomeType.values().length];
+ 
+    public BiomeLayer(boolean p_i232147_1_) {
+-      if (p_i232147_1_) {
+-         this.field_151623_c = field_202743_q;
++      for (net.minecraftforge.common.BiomeManager.BiomeType type : net.minecraftforge.common.BiomeManager.BiomeType.values()) {
++         com.google.common.collect.ImmutableList<net.minecraftforge.common.BiomeManager.BiomeEntry> biomesToAdd = net.minecraftforge.common.BiomeManager.getBiomes(type);
++         int idx = type.ordinal();
```

Comment:
You could probably get rid of this line as well.
Just use `type.ordinal()` in the couple of places you are using it.

Reply:
I am not quite sure as to what you mean @OrionDevelopment ?  Which line?

Reply:
See this line 14 of your patch file?. It defines the variable idx as type.ordinal().
And is then only used todo look ups 3 times.
This variable as such is not really needed, and can be removed to reduce the patch size.

Reply:
Ah, will remove